tutorial1_intro/demo/main.js

```diff
@@ -0,0 +1,38 @@
+// load in csv
+d3.csv("../../data/surveyResults.csv").then(data => {
+  // once the data loads, console log it
+  console.log("data", data);
+
+  // select the `table` container in the HTML
+  const table = d3.select("#d3-table");
+
+  /** HEADER */
+  const thead = table.append("thead");
+  thead
+    .append("tr")
+    .append("th")
+    .attr("colspan", "7")
+    .text("Pre-Survey Results");
```

Comment:
not sure the order you intended but I would do the rest of the code to solidify the `join` then come back to do this part, since I wouldn't want them getting lost in `colspan`. Just wanted to note that